CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 <!-- All new changelog items should come here -->
 
+- [#470]
+  - **Description:** Fix bug and add test guard in MediaQuery implementation
+  - **Products impact:** none
+  - **Addresses:** -
+  - **Components:** none
```

Comment:
Just a note that here, all affected components would ideally be included here even though it doesn't modify them directly, it's something that's used under the hood (this will help with the release notes).

Reply:
No need to do anything about it, I already prepared the release notes for this item too. Just jumping into PRs and using them as spreading the word.